app/admin/golem/jobs/page.tsx

```
@@ -0,0 +1,420 @@
+            onClick={fetchJobs}
+            className="flex items-center gap-2 rounded-lg border border-white/10 px-3 py-2 text-sm text-white/60 hover:bg-white/5 transition-colors"
+          >
+            <RefreshCw className={`h-4 w-4 ${loading ? 'animate-spin' : ''}`} />
+          </button>
+        </div>
+
+        {showFilters && (
+          <div className="flex flex-wrap gap-4 p-3 rounded-lg bg-white/5 border border-white/10">
+            <div>
+              <label className="block text-xs text-white/50 mb-1">Status</label>
+              <select
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)